src/encoded/commands/deploy.py

```diff
@@ -1,91 +1,5 @@
 """
 Encoded Application AWS Deployment Helper
-
```

Comment:
Why is this deleted here?

Reply:
Changes are from another branch: https://github.com/ENCODE-DCC/encoded/commit/b1c5663de0c2453b3abc1297778b9d0d7167aa30. The commit in the old `dev`which I rebased on was probably removed from the current `dev`, or I mistakenly rebased on `ENCD-5492-remove-cloud-config-dir` which doesn't exist anymore. We may have to rebase and code review again.